.circleci/config.yml

```diff
@@ -166,14 +166,25 @@ jobs:
           condition: <<parameters.use_review_app>>
           steps:
             - run:
-                name: Set Heroku Review Apps URL for Pull Requests
+                name: Run E2E tests (same instance)
                 command: |
-                  echo "export E2E_BASE_URL_REVIEW_APP=$(sh .circleci/get-review-app-url.sh)" >> $BASH_ENV
+                  echo "export REDIS_URL=''" >> $BASH_ENV
```

Comment:
I feel like it these are no longer dynamic then we can set them on the Job using [Circle's built-in support](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-job) for envs.

Reply:
Or they could remain dynamic… I was just getting it working. But happy to set them as you suggest

Reply:
Also true. They could come from CI still. Maybe that’s better too